syncmaster/backend/api/v1/groups.py

```diff
@@ -196,6 +214,10 @@ async def add_user_to_group(
         raise ActionNotAllowedError
 
     async with unit_of_work:
+        group = await unit_of_work.group.read_by_id(group_id=group_id)
+        if group.owner_id == user_id:
+            raise AlreadyIsGroupOwnerError
```

Comment:
```suggestion
    group = await unit_of_work.group.read_by_id(group_id=group_id)
    if group.owner_id == user_id:
        raise AlreadyIsGroupOwnerError

    async with unit_of_work:
```
nit: UnitOfWork context manager is used for changes, not for checks